charts/agent/templates/daemonset.yaml

```diff
@@ -315,6 +321,9 @@ spec:
               name: vardata-vol
             - mountPath: /host/var/run
               name: varrun-vol
+            - mountPath: /host/usr/lib64
+              name: lib64-vol
+              readOnly: true
```

Comment:
If I get this right, this is for the `agent-slim` container. Shouldn't the volume mount only be needed for `agent-kmodule` (`initContainer`)? If that's the case, this mount could be removed.

Reply:
I need to re-test this but if I recall correctly we need it for ebpf, updates soon